docs/layers.md

```diff
@@ -796,6 +796,7 @@ To improve performance, some road segments are merged at low and mid-zooms. To f
 * `aeroway`: See kind list below.
 * `ferry`: See kind list below.
 * `highway`: See kind list below.
+* `hiking_network`: Present if the road is part of a hiking network. If so, the value will be one of `iwn` for International Walking Network, `nwn` for National Walking Network, `rwn` for Regional Walking Network, `lwn` for Local Walking Network.
```

Comment:
Should we say `walking_network` or `hiking_network`?

For cycle network elsewhere, we're saying `bicycle_*` as that's the more common OSM'ism (excepting `cycle_barrier`), so it's okay to have some variation. But flagging for discussion.